apps/trading/.env

```diff
@@ -19,6 +19,7 @@ NX_DEPLOY_URL=$DEPLOY_URL
 NX_DEPLOY_PRIME_URL=$DEPLOY_PRIME_URL
 
 # App configuration variables
+NX_VEGA_ENV = "TESTNET"
 NX_VEGA_URL = "https://lb.testnet.vega.xyz/query"
 NX_ETHEREUM_CHAIN_ID = 3
-NX_INFURA_ID = "4f846e79e13f44d1b51bbd7ed9edefb8"
\ No newline at end of file
+NX_INFURA_ID = "4f846e79e13f44d1b51bbd7ed9edefb8"
```

Comment:
**praise (non-blocking)**:  This made me go and check that it doesn't need to be a secret. It doesn't! Just to save anyone else going through the same check. @mattrussell36 knows best.